Web/Presenters/GroupPresenter.php

```diff
@@ -99,6 +113,7 @@ function renderModifyAdmin(int $id): void
         $user = is_null($this->queryParam("user")) ? $this->postParam("user") : $this->queryParam("user");
         $comment = $this->postParam("comment");
         $removeComment = $this->postParam("removeComment") === "1";
+        $hidden = $this->queryParam("hidden") === "1" ? true : ($this->queryParam("hidden") === "0" ? false : null);
```

Comment:
To simplify and make this line shorter you can use this construct:
```php
$hidden = [true, false][(int) $this->queryParam("hidden")] ?? null
```

...I think.

Reply:
Due to the fact that converting a string with no number (or null) to a number will be equal to 0, this code does not perform its original function (return null if the string is not 0 or 1). By the way, for this code to convert a string to a boolean, you need to swap true and false in the array. I will use the following:
```php
$hidden = ["0" => false, "1" => true][$this->queryParam("hidden")] ?? null;
```

Reply:
Done